code/datums/controllers/process/world.dm

```diff
@@ -29,6 +29,24 @@
 	doWork()
 		last_object = "sun.calc_position"
 		sun.calc_position()
+		z2_sun.calc_position()
+		z3_sun.calc_position()
+		z4_sun.calc_position()
+		z5_sun.calc_position()
+
+		earth_outside_sun.calc_position()
+		earth_garden_sun.calc_position()
+		moon_sun.calc_position()
+		mars_1_sun.calc_position()
+		mars_2_sun.calc_position()
+		io_sun.calc_position()
+		biodome_sun.calc_position()
+		senex_sun.calc_position()
+		earth_dojo_1_sun.calc_position()
+		earth_dojo_2_sun.calc_position()
+		graveyard_sun.calc_position()
+		graveyard_swamp_sun.calc_position()
+
```

Comment:
Why not iterate over `starlist`?

Reply:
That is a really good idea that i completely did not consider. Adding that now.